lib/server.js

```diff
@@ -575,6 +575,11 @@ function addBucketsRoutes(server) {
         name: 'GetBucket'
```

Comment:
Let's delete this `GetBucket` handler to make sure we an HTTP 405 if anyone tries this operation.

Reply:
:+1: